.env.example

```
@@ -171,6 +180,7 @@ LT_HOSTNAME=translate.yourdomain.com
 N8N_HOSTNAME=n8n.yourdomain.com
 NEO4J_HOSTNAME=neo4j.yourdomain.com
 NOCODB_HOSTNAME=nocodb.yourdomain.com
+OLLAMA_HOSTNAME=ollama.yourdomain.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OLLAMA_HOSTNAME key should go before the USER_DOMAIN_NAME key

(UnorderedKey)